src/index.html

```diff
@@ -21,7 +24,373 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <h1 class="page__title">Catalog</h1>
+
+    <header class="header">
+      <div class="header__logo">
+        <a
+          class="header__link"
+          href="#"
+        >
+          <img
+            class="header__img"
+            src="./images/logo.png"
+            alt="MOYO Logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
```

Comment:
Fix code style everywhere if you have only 2 attributes
```suggestion
            <a href="#" class="nav__link">
              Samsung
            </a>
```